apps/sms/test/marionette/lib/views/dialer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var View = require('./view');
+var Dialer =  require('../../../../../../apps/communications/dialer/test/marionette/lib/dialer.js');
+
+function DialerWrapper(client) {
+  this.client = client;
+}
+module.exports = DialerWrapper;
+
+DialerWrapper.prototype = {
+  __proto__: View.prototype,
+
+  switchToFrame: function() {
```

Comment:
I understand. However, I don't think comfortable with renaming it to `switchToDialerFrame()`, because we'd use it as `dialer.switchToDialerFrame()`, and we'd have redundant information.

Do you think `dialer.switchToFrame()` doesn't imply it switches to the top-level frame of the dialer?